test/bdd/features/wallet_jsonld.feature

```diff
@@ -0,0 +1,31 @@
+#
+# Copyright SecureKey Technologies Inc. All Rights Reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+@wallet_jsonld
+Feature: Support for JSON-LD in universal wallet
+
+  Scenario Outline: Universal wallet JSON-LD support
```

Comment:
steps should be like,

- Create Wallet profile with some settings for holder ZZZ
- open wallet
- XXX issues credential YYY....
- holder ZZZ saves credential YYY into his wallet
-...similarly verifier steps follow


Remember: Issuer/Verifier never uses wallet, wallet is used by holder only.

Sign/Verify steps by issuer/verifier here, should never use universal wallet.

Note: Universal wallet has sign/verify functions, but they are for wallet's internal use

Reply:
Updated, please see latest changes.